src/client.cpp

```diff
@@ -743,6 +745,20 @@ bool Client::loadMedia(const std::string &data, const std::string &filename)
 		return true;
 	}
 
+	const char *script_ext[] = {
+		".lua",
+		NULL
+	};
+	name = removeStringEnd(filename, script_ext);
+	if(name != "")
```

Comment:
Bad code style shouldn't be adapted. [Guidelines](http://dev.minetest.net/Code_style_guidelines)